cdap-app-fabric/src/main/java/io/cdap/cdap/app/preview/PreviewStatus.java

```diff
@@ -16,9 +16,9 @@
 
 package io.cdap.cdap.app.preview;
 
+import com.google.common.base.Objects;
```

Comment:
It's better to use the `java.util.Objects` then the one from guava.

Reply:
done.